### PR TITLE
Remove the use of the BigInt type (#438)

### PR DESCRIPTION
**Pull request by hbenl**

### CODE REVIEW

**Comment by hbenl** [comment]
see also https://github.com/RecordReplay/devtools/pull/687#issuecomment-695797443

File: src/devtools/client/webconsole/reducers/messages.js [hunk 1]
```diff
@@ -1403,6 +1403,21 @@ function maybeSortVisibleMessages(state, sortWarningGroupMessage = false, timeSt
...
+        return 1;
+      } else {
+        const _a = messageCountSinceLastExecutionPoint(state, a);
+        const _b = messageCountSinceLastExecutionPoint(state, b);
+        return _a < _b ? -1 : _a > _b ? 1 : 0;
+      }
+    });
     state.visibleMessages = sortBy(
       state.visibleMessages,
       message => BigInt(messageExecutionPoint(state, message)),
```
Comment (bhackett1024) [inline src/devtools/client/webconsole/reducers/messages.js:1423]: This sorting code still needs to be removed.

File: src/protocol/utils.js [hunk 2]
```diff
@@ -144,6 +144,10 @@ ArrayMap.prototype = {
   },
 };
 
+function compareBigInt(a, b) {
```
Comment (bhackett1024) [inline src/protocol/utils.js:147]: Can this be compareNumericStrings or something, with a comment describing what's happening?

**Review by bhackett1024 — CHANGES_REQUESTED** [review]

File: src/devtools/client/webconsole/reducers/messages.js (hunk 1 above)
Comment (hbenl) [reply src/devtools/client/webconsole/reducers/messages.js:1423]: D'oh!

File: src/protocol/utils.js (hunk 2 above)
Comment (hbenl) [reply src/protocol/utils.js:147]: I like compareNumericStrings

**Review by bhackett1024 — APPROVED** [review]
Thanks!

**Pull request merged**